sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -164,3 +163,17 @@ case class Uuid(randomSeed: Option[Long] = None) extends LeafExpression with Sta
 
   override def freshCopy(): Uuid = Uuid(randomSeed)
 }
+
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage =
+    """_FUNC_() - Returns the Spark version. The string contains 2 fields, the first being a release or snapshot version and the second being a git revision.""",
+  since = """3.0.0""")
```

Comment:
Can you add an example for tests?

Reply:
I am afraid example here only take literals as result candidates